sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/AbstractSentryWebFilter.java

```diff
@@ -68,19 +68,13 @@ protected void doFinally(
     if (transaction != null) {
       finishTransaction(serverWebExchange, transaction);
     }
-    if (requestHub.isEnabled()) {
```

Comment:
Shouldn't be needed since we create a new fork from root scopes for each request.